rerun_py/docs/gen_common_index.py

```diff
@@ -82,6 +82,12 @@ class Section:
         func_list=["log", "set_time_sequence", "set_time_seconds", "set_time_nanos"],
         class_list=[],
     ),
+    Section(
+        title="Adhoc Data",
```

Comment:
"Any", "Adhoc", … we should have one coherent name for this. "Custom Data" is another. I don't know

Reply:
I like Custom.